src/Libraries/CoreNodeModels/DynamoConvert.cs

```diff
@@ -86,6 +91,23 @@ public ConversionUnit SelectedToConversion
             }
         }
 
+        /// <summary>
+        /// Conversion factor of from unit and to to unit.
+        /// </summary>
+        [JsonProperty("ConversionFactor")]
```

Comment:
@ke-yu Your previous commit looks to be correct here except for a copy/paste error with both convert from and convert to having the same json property name.